No query found

### Describe the bug

My initial run of the cell with `quak.widget`works well but the second one gives me the following error message:
```
Uncaught AssertionError: No query found for 738ad943-6150-4bfc-a356-3b84be0e2be9
    at vt (554739-16008-0RQMdqAm.js:4:17497)
    at 554739-16008-0RQMdqAm.js:265:2701
    at index-Cg7PcY_l.js:80:505821
    at Set.forEach (<anonymous>)
    at dZe.receiveCustomMessage (index-Cg7PcY_l.js:80:505809)
    at Object.d [as current] (index-Cg7PcY_l.js:80:508377)
    at t.a (index-Cg7PcY_l.js:32:139152)
    at index-Cg7PcY_l.js:80:98468
    at Set.forEach (<anonymous>)
    at HTe.broadcastMessage (index-Cg7PcY_l.js:80:98453)
```

### Environment

<details>

```
{
  "marimo": "0.11.19",
  "OS": "Windows",
  "OS Version": "11",
  "Processor": "AMD64 Family 25 Model 116 Stepping 1, AuthenticAMD",
  "Python Version": "3.13.2",
  "Binaries": {
    "Browser": "134.0.6998.89",
    "Node": "v22.14.0"
  },
  "Dependencies": {
    "click": "8.1.8",
    "docutils": "0.21.2",
    "itsdangerous": "2.2.0",
    "jedi": "0.19.2",
    "markdown": "3.7",
    "narwhals": "1.30.0",
    "packaging": "24.2",
    "psutil": "7.0.0",
    "pygments": "2.19.1",
    "pymdown-extensions": "10.14.3",
    "pyyaml": "6.0.2",
    "ruff": "0.9.10",
    "starlette": "0.46.1",
    "tomlkit": "0.13.2",
    "typing-extensions": "4.12.2",
    "uvicorn": "0.34.0",
    "websockets": "15.0.1"
  },
  "Optional Dependencies": {
    "altair": "5.5.0",
    "anywidget": "0.9.15",
    "duckdb": "1.3.0.dev926",
    "pandas": "2.2.3",
    "polars": "1.24.0",
    "pyarrow": "19.0.1"
  },
  "Experimental Flags": {}
}
```

</details>


### Code to reproduce

```python
import polars as pl
import quak
df = pl.read_parquet("https://github.com/uwdata/mosaic/raw/main/data/athletes.parquet")
widget = quak.Widget(df)
widget
```

--- a/frontend/src/plugins/impl/anywidget/AnyWidgetPlugin.tsx
+++ b/frontend/src/plugins/impl/anywidget/AnyWidgetPlugin.tsx
@@ -129,7 +129,15 @@ const AnyWidgetSlot = (props: Props) => {
   }
 
   return (
-    <LoadedSlot {...props} widget={module.default} value={valueWithBuffer} />
+    <LoadedSlot
+      // Use the jsUrl as a key to force a re-render when the jsUrl changes
+      // Plugins may be stateful and we cannot make assumptions that we won't be
+      // so it is safer to just re-render.
+      key={jsUrl}
+      {...props}
+      widget={module.default}
+      value={valueWithBuffer}
+    />
   );
 };
 

